src/beast/site_scons/site_tools/VSProject.py

```diff
@@ -127,6 +127,11 @@ def gen():
             if type(item) == dict:
                 for k, v in xsorted(item.items()):
                     yield k + '=' + v
+            elif type(item) == tuple:
```

Comment:
This could be improved a tiny bit.

First, this catches tuples - but not classes derived from tuple, which is perfectly legal.  The code might not be doing that today, but it might later.

Same comment about list.

Better code might be `elif isinstance(item, (list, tuple)):`